spec/requests/drafts_controller_spec.rb

```diff
@@ -24,4 +24,15 @@
     parsed = JSON.parse(response.body)
     expect(parsed["drafts"].length).to eq(0)
   end
+
+  it 'does not let a user see drafts stream of another user' do
+    user_b = Fabricate(:user)
+    Draft.set(user_b, 'xxx', 0, '{}')
+
+    sign_in(Fabricate(:user))
+    get "/drafts.json", params: { username: user_b.username }
+    expect(response.status).to eq(200)
```

Comment:
I'm thinking if we should return a `200` response code here since the `username` param is invalid if not signed in as the current user.

Reply:
Would `raise Discourse::InvalidAccess` be appropriate here? It would also mean that in the frontend, users attempting `/u/user_b/activity/drafts` would get an Access Denied screen. 

Seems reasonable to me @tgxworld.